docs/tutorials/creating-a-runtime-module.md

```diff
@@ -2,46 +2,68 @@
 title: "Creating a Runtime Module"
 ---
 
-In this tutorial, you'll write a Substrate runtime module that lives in its own crate, and include it in a node based on the node-template.
+In this tutorial, you'll write a Substrate runtime module that lives in its own
+crate, and include it in a node based on the node-template.
 
 ## Setup Your Development Environment
 
-If you haven't already, follow the [fast installation](getting-started/installing-substrate.md#fast-installation) for getting started on Substrate. It will provide you with all the development resources you need to build Substrate and the [`substrate-up` scripts](getting-started/using-the-substrate-scripts) for quickly setting up new nodes.
+If you haven't already, follow the
+[fast installation](getting-started/installing-substrate#fast-installation) for
```

Comment:
We have a new getting-started page which doesnt have an "installing-substrate" sub page.